testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -107,6 +108,14 @@ pub struct EmitJobRequest {
     pub thread_params: EmitThreadParams,
 }
 
+pub static GAS_UNIT_PRICE: Lazy<u64> = Lazy::new(|| {
+    if let Ok(v) = env::var("GAS_UNIT_PRICE") {
+        v.parse().expect("Failed to parse GAS_UNIT_PRICE")
+    } else {
+        0_u64
```

Comment:
Can we set default to `1`?

Reply:
observed 20% to 30% perf regression if we default gas_price = 1, just set it back to 0 and we can change this in future. Good news is I confirmed with platform team which this perf regression is expected